src/nylo/tokens/value.py

```diff
@@ -1,65 +1,90 @@
+# This file is a part of nylo
```

Comment:
value.py should not have the copyright. Only file beggining with __ shall have them.